site/_apps/nextjs-hn.txt

```diff
@@ -0,0 +1,26 @@
+---
+weight: 14
+title:  'Next.js HN'
+github-title: 'chrisdwheatley/nextjs-hn'
+libraries:
+    - name: 'Next.js'
+    - name: 'Preact'
+module-bundling: 'Next.js with additional Webpack config'
+service-worker: 'Application & data caching with sw-precache'
+perfomance-patterns: 'In memory LRU cache, DNS prefetch, Preact compat alias in production'
+server-side-rendering: 'Yes'
+api: 'Node-hnapi (unofficial)'
+hosting: 'Digital Ocean & Cloudflare'
+authors:
+    - name: 'Chris'
+lighthouse: '91/100'
+interactive-em: '4.7s'
+interactive-faster-3g: '3.0s'
+lighthouse-link: https://www.webpagetest.org/result/170607_PW_1d454612c736a65ddff058cf16b8842e/
+wpt-em-link: https://www.webpagetest.org/result/170607_PW_1d454612c736a65ddff058cf16b8842e/
+wpt-faster-3g-link: https://www.webpagetest.org/result/170607_DY_400d1cc5db11ec9727404dc9cbc441e6/
```

Comment:
Sigh. That run also failed due to main thread issues. Trying again:

https://www.webpagetest.org/result/170608_DD_b9e141e3eb2598266ac4f7d55a1321ef/